sale_channel_hook_delivery_done/tests/test_hook_delivery_done.py

```diff
@@ -40,10 +43,10 @@ def test_hook_delivery_done(self):
             lambda r: r.picking_type_id == self.env.ref("stock.picking_type_out")
         )
         picking_pick = self.sale.picking_ids - picking_ship
-        picking_pick.move_lines.move_line_ids.qty_done = 1.00
+        picking_pick.move_lines.quantity_done = 1.00
```

Comment:
quantity_done is a computed field, that depends on move_line_ids.qty_done. I think leaving it on qty_done better represents a real world scenario with real data

Reply:
the compute field quantity_done is defined with an inverse fct _quantity_done_set